src/lib/api/containerApiClient.test.ts

```diff
@@ -0,0 +1,96 @@
+import { fetchAllContainerList, fetchContainerList } from '@/lib/api';
+import { request } from '@/lib/api/commonUtils';
+
+jest.mock('@/lib/api/commonUtils', () => ({
+  request: jest.fn(),
+}));
+
+const mockContainer = {
+  id: '939a4c50-ee7b-4558-b8d5-371514e77bed',
+  name: 'Shared-houses Fridge',
+  group: { groupId: 'a3kdifut-a520-c2cb-1be7-d90710691861', groupName: 'Shared-house' },
+  foods: [
+    {
+      id: '11111111-58b7-4c3d-9548-fc0a4d3b3ef4',
+      name: 'arugula',
+      quantity: 500,
+      category: 'vegetables',
+      unit: 'g',
+      expiry: '2023-12-10T00:00:00.000Z',
+      createdAt: '2023-12-01T00:00:00.000Z',
+    },
+  ],
+};
+
+/**
+ * Create mock data for request method
+ * @template T
+ * @param {T} mockData
+ * @return {*} mocked request method
```

Comment:
I suppose that `*` means `any` type; however, in this case, it should be `jest.MockedFunction<typeof request>` or something like that, to be accurate.

Reply:
I fixed the code. Thanks!